### PR TITLE
Add documentation for sites.deleted_at

### PR DESCRIPTION
**Pull request by curtis1000**

### CODE REVIEW

File: source/methods/sites.md.erb [hunk 1]
```diff
@@ -41,6 +41,13 @@ $result = $wiw->get("sites");
 ### HTTP Request
 `GET <%=@api_prefix%>/2/sites`
 
+`GET <%=@api_prefix%>/2/sites?include_deleted=true`
+
+### Parameters
+
+Key | Description
+--- | -----------
+include_deleted | boolean query string param to include deleted sites. Optional, defaults to false.
```
Comment (shadowhand) [inline source/methods/sites.md.erb]: We have a nice `param` formatter. Let's use it.

**Comment by shadowhand** [comment]
Has this change been shipped? I'm not sure what PR it was associated with.

**Comment by curtis1000** [comment]
No, it's actually 2 PRs. 

We need to get the more generic ```ORM::soft_delete()``` one merged:
https://github.com/wheniwork/wheniwork-app/pull/2870

And then I would need to rebase the ```sites.deleted_at``` one based on the one above:
https://github.com/wheniwork/wheniwork-app/pull/2368

**Comment by shadowhand** [comment]
This needs a rebase.

**Comment by shadowhand** [comment]
@curtis1000 can you rebase this please?

**Comment by curtis1000** [comment]
@shadowhand done

**Comment by shadowhand** [comment]
@curtis1000 thx!

**Pull request merged**